### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by theqingli**
Added IDbyDNA

Adds the following events or companies:
 - 
 - 

### Checklist

 - [] I have updated the event or company counts
 - [] I have put the most recent relevant date in the "Last Update" column
 - [] I have linked to the article about the change, not the company's website

### CODE REVIEW

**Comment by theqingli** [comment]
I only added one line about my company "IDbyDNA". What do I need to do to get the update to the web page? Thanks.

File: README.md [hunk 1]
```diff
@@ -73,7 +73,6 @@ This is the running list of what in tech has been affected by COVID-19. Pull req
 | Google[[1]](https://www.theguardian.com/world/2020/mar/04/coronavirus-google-tech-dublin-twitter-work-from-home#maincontent) [[2]](https://www.reuters.com/article/us-health-coronavirus-alphabet/google-recommends-all-north-america-employees-to-work-from-home-idUSKBN20X38J?feedType=RSS&feedName=technologyNews) | Encouraged / Geo-specific | Restricted | Restricted | ? | 2020-03-11 |
 | Granular.ag | Required for SF / All others allowed | Restricted | Restricted for SF | Restricted | 2020-03-05 |
 | Groupon | Encouraged | Restricted | ? | Restricted | 2020-03-06 |
-| HASH | Encouraged | Restricted | Restricted | Restricted | 2020-03-11 |
```
Comment (drewbrew) [inline README.md]: This line needs to go back in.

File: README.md [hunk 2]
```diff
@@ -120,7 +120,6 @@ This is the running list of what in tech has been affected by COVID-19. Pull req
 | Parler | Required | Restricted | Restricted | Restricted | 2020-03-06 |
 | Patreon | Required | Restricted | Restricted | ? | 2020-03-11 |
 | Philo | Required | Restricted | Restricted | Restricted | 2020-03-11 |
-| Pilates Anytime | Encouraged | Restricted | Restricted | Restricted | 2020-03-11 |
```
Comment (drewbrew) [inline README.md]: This line needs to go back in as well.

File: README.md [hunk 3]
```diff
@@ -170,6 +169,7 @@ This is the running list of what in tech has been affected by COVID-19. Pull req
 | Zeus Living | Required | Restricted | Restricted | Restricted | 2020-03-10 |
 | Zillow | Encouraged | Restricted | ? | ? | 2020-03-06 |
 | Zulily | Encouraged | Restricted | Restricted | ? | 2020-03-07 |
+| HASH | Encouraged | Restricted | Restricted | ? | 2020-03-11 |
```
Comment (drewbrew) [inline README.md]: This line needs to be deleted (was moved up higher to fix sorting)

**Pull request closed without merging**